corehq/apps/api/tests.py

```diff
@@ -966,12 +988,22 @@ def __init__(self, objs):
     def obj_get_list(self):
         return self.objs
 
+    def detail_uri_kwargs(self, bundle_or_obj):
+        return {
+            'pk': get_obj(bundle_or_obj).other_model_id
```

Comment:
I see - it's a field on this test model.  I thought it was some really awkwardly named couchdbkit api I hadn't seen before.